core/src/main/scala/filodb.core/query/RangeVector.scala

```diff
@@ -471,7 +471,7 @@ object SerializedRangeVector extends StrictLogging {
       // If there weren't containers before, then take all of them.  If there were, discard earlier ones, just
       // start with the most recent one we started adding to
       val containers = oldContainerOpt match {
-        case None => builder.allContainers
+        case None => builder.allContainers.toList
```

Comment:
Please add detailed explanation on why this is needed

Reply:
Done